scripts/minimize-runtime-dependencies--linux.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+#
+# This script *can* help in minimizing runtime dependencies of
+# Daedalus… but only the ones which prevent Electron from starting.
+#
+# That’s a good starting point before we get this automatically from
+# webpack – <https://input-output.atlassian.net/browse/DDW-1103>.
+#
```

Comment:
I used this script to quickly find missing installer dependencies, perhaps it should be removed from here, and added to `webpack-bundle-analyzer` PR by @szymonmaslowski:
* https://input-output.atlassian.net/browse/DDW-1103
* commit: https://github.com/input-output-hk/daedalus/commit/6b8e610b33ac94ea7d987c34c70a1661bd4cd508
* branch: https://github.com/input-output-hk/daedalus/tree/webpack-bundle-analyzer

Reply:
@michalrus what's the status on this?

Reply:
This will be very elegantly superseded by @szymonmaslowski’s https://github.com/input-output-hk/daedalus/pull/2997 :bow: 

Thanks for catching, I’ll drop this file from here! :bow: